controllers/storagecluster/external_resources.go

```diff
@@ -285,6 +285,13 @@ func (obj *ocsExternalResources) ensureCreated(r *StorageClusterReconciler, inst
 func (obj *ocsExternalResources) ensureDeleted(r *StorageClusterReconciler, instance *ocsv1.StorageCluster) (reconcile.Result, error) {
 
 	if isExternalOCSProvider(instance) {
+
+		// make sure consumer is onboarded before offboarding
```

Comment:
Comment can be a bit confusing.
```suggestion
		// skip offboarding if consumer is not on onboarded
```